drivers/network/dd/e1000/info.c

```diff
@@ -239,6 +239,12 @@ MiniportQueryInformation(
         break;
     }
 
+    case OID_GEN_PHYSICAL_MEDIUM:
```

Comment:
This needs to be added in the `SupportedOidList` array above, as well.

Reply:
Done